css/index.css

```diff
@@ -0,0 +1,309 @@
+body {
+  background-color: #f2f6f9;
+  margin: 0;
+  font-family: "Poppins";
+  font-weight: 500;
+}
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  background-color: #f2f6f9;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.btn-back {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 4px 8px;
+  border: 1px solid #081c3933;
+  border-radius: 5px;
+  width: 73px;
+  height: 32px;
+  font-family: "Poppins", sans-serif;
+  font-size: 14px;
+  background-color: #f2f6f9;
+  position: absolute;
+  top: 32px;
+  left: 32px;
+}
+
+.img {
+  width: 16px;
+  height: 24px;
+  margin-right: 6px;
+}
+
+.title {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  margin-top: 28px;
+  font-weight: 600;
+  font-size: 48px;
+  line-height: 72px;
+  margin-bottom: 32px;
+  color: #0C2447;
+}
+
+.subtitle {
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #203a62;
+}
+
+.switch-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  margin-top: 40px;
+}
+
+.monthly {
+  font-size: 16px;
+  line-height: 24px;
+  color: #203a62;
+  margin-right: 12px;
+}
+
+.img-switch {
+  margin-right: 12px;
+}
+
+.annually {
+  font-size: 16px;
+  line-height: 24px;
+  color: #6f809a;
+  margin-right: 12px;
+}
+
+.save {
+  font-size: 14px;
+  line-height: 21px;
+  color: #74ca77;
+}
+
+.main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 32px auto 0 auto;
+  column-gap: 20px;
+  row-gap: 50px;
+  flex-wrap: wrap;
+  padding-bottom: 80px;
+  box-sizing: border-box;
+}
+
+.plan {
+  width: 320px;
+  height: 593px;
+  background: #ffffff;
+  box-shadow: 0px 4px 6px rgba(12, 36, 71, 0.07);
+  border-radius: 17px;
+  padding: 32px;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.img1 {
+  margin-right: 16px;
+}
+
+.block-header {
+  display: flex;
+}
+
+.header-text {
+  display: flex;
+  flex-direction: column;
+}
+.plan-name {
+  font-family: "Poppins";
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 32px;
+  margin-bottom: 9px;
+}
+
+.dollar {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 36px;
+  color: rgba(12, 36, 71, 0.3);
+}
+
+.number {
+  font-family: "Poppins";
+  font-size: 32px;
+  line-height: 48px;
+  font-weight: 700;
+  font-style: normal;
+}
+
+.month {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 36px;
+  color: rgba(12, 36, 71, 0.3);
+}
+.info-container {
+  margin-top: 24px;
+  display: flex;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.note {
+  margin-right: 8px;
+}
+.songs {
+  font-family: "Poppins";
```

Comment:
looks like family+style+others can be one selector https://developer.mozilla.org/ru/docs/Web/CSS/font

Reply:
+ check it for others styles